e2e/jest.test-setup.ts

```diff
@@ -88,13 +95,17 @@ beforeEach(async () => {
     if (!message.args().length) return;
     const title = await getPageTitle();
     try {
-      const args = await Promise.all(message.args().map((a) => describeJsHandle(a)));
+      const args = await Promise.all(message.args().map((jsHandle) => describeJsHandle(jsHandle)));
+      let concatenatedText = '';
+      for (let i = 0; i < args.length; ++i) {
+        if (args[i] !== undefined) {
+          concatenatedText += `${args[i]}\n`;
+        }
+      }
```

Comment:
Slight change in how we retrieve browser Console messages. There were some Console messages are not logged by the old way.